scripts/talos/index.test.js

```diff
@@ -60,10 +63,16 @@ describe('Talos', () => {
 
     expect(runNpmInstall).toHaveBeenCalledTimes(2);
 
-    expect(bumpChangelogs).toHaveBeenCalledTimes(1);
+    expect(bumpChangelogs).toHaveBeenCalledTimes(2);
     expect(bumpChangelogs).toHaveBeenCalledWith({
-      changesDescription: 'Talos - Bump Dependencies',
-      packageNames: ['apple', 'pears'],
+      changesDescription: 'Talos - Bump Dependencies - package1  ^1.2  →  ^1.4',
+      packageNames: ['apple'],
+      prLink: 'prURL',
+    });
+    expect(bumpChangelogs).toHaveBeenCalledWith({
+      changesDescription:
+        'Talos - Bump Dependencies - package1  ^1.2  →  ^1.4, package2  ^1.0  →  ^1.1',
```

Comment:
Just wondering if this might be too long for some possible changes maybe?

Reply:
I was wondering about that too, but it is very descriptive just by looking at the CHANGELOG

Reply:
I dont think it needs to have that much detail, it makes it hard to read quickly, IMO something like the top line would be better.
```md
| 3.1.7 | [PR#123](https://github.com/bbc/psammead/pull/1993) Talos - Bump @bbc/gel-foundations, @bbc/psammead-styles |
| 3.1.7 | [PR#123](https://github.com/bbc/psammead/pull/1993) Talos - Bump Dependencies - @bbc/gel-foundations  ^1.3.3  →  ^3.3.3, @bbc/psammead-styles  ^1.1.3  →  ^2.1.3 |
```

Please dont remove the version collection logic though, we can have it in the PR body, just filter it out of the changelog message

Reply:
Ok. Fixed!